modules/registrars/openprovider/Controllers/Hooks/AdminAreaHeaderOutputController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace OpenProvider\WhmcsRegistrar\Controllers\Hooks;
+
+use WHMCS\Database\Capsule;
+
+class AdminAreaHeaderOutputController
+{
+    public function output($vars)
+    {       
+        $return = '';
+        if ($vars["pagetitle"] == 'TLD Import & Pricing Sync') {
+            
+            $isTestMode = false;
+            $rows = Capsule::table('tblregistrars')->get();
```

Comment:
may be better to use meaningful variable name

Reply:
Changed as `$testMode`

Reply:
@Uditha-05 I asked regarding the variable `$row`.
BTW `isTestMode` is good for me

Reply:
Done!